heft-plugins/heft-webpack4-plugin/src/shared.ts

```diff
@@ -69,9 +69,9 @@ export type IWebpackConfiguration = IWebpackConfigurationWithDevServer | IWebpac
 export interface IWebpackPluginAccessorHooks {
   /**
    * A hook that allows for loading custom configurations used by the Webpack
-   * plugin. If a webpack configuration is provided, this will be populated automatically
-   * with the exports of the config file. If a webpack configuration is not provided,
-   * one will be loaded by the Webpack plugin.
+   * plugin. If a tap returns a value other than `undefined` before stage `STAGE_LOAD_LOCAL_CONFIG`,
+   * it will suppress loading from the webpack config file. To provide a fallback behavior in the
+   * absence of a local config file, tap this hook with a `stage` value greater than `STAGE_LOAD_LOCAL_CONFIG`.
```

Comment:
```suggestion
   * plugin. If a tap returns a value other than `undefined` before stage {@link STAGE_LOAD_LOCAL_CONFIG},
   * it will suppress loading from the webpack config file. To provide a fallback behavior in the
   * absence of a local config file, tap this hook with a `stage` value greater than {@link STAGE_LOAD_LOCAL_CONFIG}.
```